.github/workflows/main.yml

```diff
@@ -23,6 +23,28 @@ on:
       - github-actions
 
 jobs:
+  single_checks:
+    name: "Single sanity check"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install latest DMD
+        uses: dlang-community/setup-dlang@v1
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Run tests
+        run: |
+          # check for trailing whitespace
+          TRAILING_WS_COUNT=$(find . -type f -name '*.d' -exec grep -Hn "[[:blank:]]$" {} \; | wc -l)
```

Comment:
This should be replaced by:
https://github.com/marketplace/actions/editorconfig-action